packages/guides-restructured-text/src/RestructuredText/Parser/Productions/ListRule.php

```diff
@@ -70,11 +71,11 @@ public function apply(DocumentParserContext $documentParserContext, CompoundNode
     {
         $documentIterator = $documentParserContext->getDocumentIterator();
 
-        $buffer = new Buffer();
+        $buffer = new Buffer(unindentStrategy: UnindentStrategy::FIRST);
         //First line sets the listmarker of the list, and the indentation of the current item.
         $listConfig = $this->getItemConfig($documentIterator->current());
         if (trim($documentIterator->current()) !== $listConfig['marker']) {
-            $buffer->push(mb_substr($documentIterator->current(), $listConfig['indenting']));
```

Comment:
I'm not sure this change is valid, right now the first list item defined the level op indentation, so this could never be a fixed value. 

However I think we could just push the raw content in the buffer and apply the stripping like on the other blocks?